src/ClusterBootstrap/services/job-insighter/job-insighter-cronjob.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: job-insighter-cronjob
+  labels:
+    app: job-insighter
+spec:
+  concurrencyPolicy: Forbid
+  jobTemplate:
+    metadata:
+      name: job-insighter-job
+      labels:
+        app: job-insighter
+    spec:
+      activeDeadlineSeconds: 900
+      template:
+        metadata:
+          name: job-insighter-pod
+          labels:
+            app: job-insighter
+        spec:
+          containers:
+          - name: job-insighter
+            image: '{{cnf["worker-dockerregistry"]}}/{{cnf["dockerprefix"]}}/job-insighter:{{cnf["dockertag"]}}'
+            imagePullPolicy: Always
+            # Job insighter requires that restful server and prometheus run on the infra node.
+            args:
+              - 'python'
+              - '/job-insighter/insight.py'
+              - '--restful_url'
+              - 'http://localhost:{{cnf["restfulapiport"]}}'
+              - '--prometheus_url'
+              - 'http://localhost:{{cnf["prometheus"]["port"]}}'
+          {% if cnf["private_docker_registry_username"] %}
+          imagePullSecrets:
+          - name: svccred
+          {% endif %}
+          nodeSelector:
+            job-insighter: active
+          hostNetwork: true

```

Comment:
I think we do not need host network?

Reply:
I am using `localhost` in calls to restfulapi and prometheus. Without host network, it doesn't work.